docs/explanation/security.md

```diff
@@ -0,0 +1,58 @@
+# Security
+
+This document outlines common risks and possible best practices specifically for the DNS charms. It
+focuses on configurations and protections available through the charms themselves.
+
+The overall best practice is to [keep your charms updated](https://documentation.ubuntu.com/juju/3.6/reference/juju-cli/list-of-juju-cli-commands/refresh/) to the latest version available.  
+A good understanding of [the DNS system](https://bind9.readthedocs.io/en/stable/chapter1.html) is also helpful.
+
+## Configuration
+
+The DNS charms have security-related configurations, and misconfiguring them can lead to vulnerabilities.  
+
+### bind-operator
+
+No security-related configurations available.
+
+### dns-policy
+
+`dns-policy` uses a Django application under the hood for its API and web interface. As such, it can be configured following
```

Comment:
What about adding advice on how to store the user/password for accessing the application, and what could happen if someone with bad intentions gains access to it?

Reply:
Added a paragraph to explain a bit more about reviewers and the security implications of these users.